### PR TITLE
cpu.py:add new model EPYC-Milan and delete old model

### PR DESCRIPTION
**Pull request by nanliu-r**
ID: 1939867

Signed-off-by: Nana Liu <nanliu@redhat.com>

### CODE REVIEW

**Comment by nanliu-r** [comment]
Test on AMD Milan host PASS:
 (1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.boot: PASS (171.53 s)

QEMU CLI:  -cpu 'EPYC-Milan',+kvm_pv_unhalt

**Comment by nanliu-r** [comment]
@vivianQizhu @huangyum please help to review this, thanks.

File: virttest/cpu.py [hunk 1]
```diff
@@ -50,15 +50,15 @@
                               "Haswell", "Haswell-noTSX", "IvyBridge",
                               "SandyBridge", "Westmere", "Nehalem",
                               "Penryn", "Conroe"]}
-CPU_TYPES_RE = {"EPYC-Rome": "rdpid,wbnoinvd,stibp,clwb,umip",
+CPU_TYPES_RE = {"EPYC-Milan": "ibrs,pcid,ssbd,erms,fsrm,invpcid,pku",
```
Comment (yanan-fu) [inline virttest/cpu.py:53]: From libvirt side, EPYC-Milan have 2 another flags `svme-addr-chk` and `amd-ssbd` when compare with EPYC-Rome,  do we need them ?

Comment (nanliu-r) [reply virttest/cpu.py:53]: > From libvirt side, EPYC-Milan have 2 another flags `svme-addr-chk` and `amd-ssbd` when compare with EPYC-Rome, do we need them ?

Yes, but svme-addr-chk can't find on host "Flags", so we won't use it to check.

Comment (yanan-fu) [reply virttest/cpu.py:53]: Thanks for your detailed info offline, got it now.

**Review by yanan-fu — APPROVED** [review]
Acked-by: Yanan Fu <yfu@redhat.com>

**Review by huangyum — APPROVED** [review]
LGTM, ACK.

**Pull request merged**